app/src/main/java/ml/docilealligator/infinityforreddit/activities/ViewPostDetailActivity.java

```diff
@@ -452,48 +505,56 @@ public void showRemovedComment(Comment comment, int position) {
 
     public void saveComment(@NonNull Comment comment, int position) {
```

Comment:
Ideally it would be in a ViewModel/repository, but there's nothing you can do right now about it